site/docs/numbers/porting/updatePortin.mdx

```diff
@@ -0,0 +1,233 @@
+
+If the order ProcessingStatus is `DRAFT`, the rules about what can be changed are much more relaxed. Validation is performed when the ProcessingStatus is changed from `DRAFT` to `SUBMITTED`. The AltSpid element can be modified if it is not configured at the system level.
+Regarding the fields which can be updated for different port types check the request schema section of [PUT /portins/{orderId}](/apis/numbers/#operation/UpdatePortin) specification.
+
+The general approach to handling this API call is to replace the elements included in the request body, and leave other preexisting elements in an unmodified condition. This is typical of a PATCH method, but because of our commitment to backwards compatibility we have elected not to "Fix" this behavior. As a result, there are some elements that cannot be modified using the <Highlight color="#9a59c5">PUT</Highlight> method. The elements affected vary by Port-In Order type, which can be determined using a <Highlight color="#00bf8c">GET</Highlight> request to the `/portins/{orderId}` endpoint.
+
+### Examples
+To update Port-in request, you must make a <Highlight color="#9a59c5">PUT</Highlight> request to our [API /portins/{orderId}](/apis/numbers/#operation/UpdatePortin) endpoint. This can be done using tools like Postman or cURL.
+<Highlight color="#9a59c5">PUT</Highlight> <code>https://dashboard.bandwidth.com/api/accounts/{accountId}/portins/{orderId}</code>
+
+<Tabs
+    groupId="updatePortin"
+    defaultValue="payload"
+    values={[
+        { label: 'Request Payload', value: 'payload', },
+        { label: 'cURL', value: 'curl', },
+    ]
+    }>
+<TabItem value="payload">
+
+```xml
+PUT https://dashboard.bandwidth.com/api/accounts/{accountId}/portins/d28b36f7-fa96-49eb-9556-a40fca49f7c6 HTTP/1.1
+Content-Type: application/xml; charset=utf-8
+Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
+
+<?xml version="1.0" encoding="UTF-8"?>
+<LnpOrderSupp>
+    <CustomerOrderId>SJM00002</CustomerOrderId>
+    <RequestedFocDate>2014-12-04T13:00:00.000Z</RequestedFocDate>
+    <BillingTelephoneNumber>8045030092</BillingTelephoneNumber>
+    <NewBillingTelephoneNumber>9175131245</NewBillingTelephoneNumber>
+    <WirelessInfo>
+        <AccountNumber>23453245</AccountNumber>
+        <PinNumber>1111</PinNumber>
+    </WirelessInfo>
+    <Subscriber>
+        <SubscriberType>RESIDENTIAL</SubscriberType>
+        <FirstName>TestName</FirstName>
+        <LastName>TestLastName</LastName>
+        <ServiceAddress>
+            <!-- If true will overwrite all address fields,
+             those which should be required remain required.
+             Affect only supp. Default false. -->
+            <ResetAddressFields>true</ResetAddressFields>
+            <HouseNumber>115</HouseNumber>
+            <StreetName>Monarch Way</StreetName>
+            <City>Cary</City>
+            <StateCode>NC</StateCode>
+            <Zip>27518</Zip>
+        </ServiceAddress>
```

Comment:
Change instances in the examples of "115 Monarch Way" to "123 Main Street" as the former is a real address.